templates/home/home-en.html

```diff
@@ -65,48 +65,28 @@ <h3><a href="https://www.canada.ca/en/services/business.html">Business and indus
 				<h3><a href="https://www.canada.ca/en/services/benefits.html">Benefits</a></h3>
 				<p>EI, family and sickness leave, pensions, housing, student aid, disabilities</p>
 			</div>
-			<div class="col-lg-4 col-md-6">
-				<h3><a href="https://www.canada.ca/en/services/health.html">Health</a></h3>
-				<p>Food, nutrition, diseases, vaccines, drugs, product safety and recalls</p>
-			</div>
 			<div class="col-lg-4 col-md-6">
 				<h3><a href="https://www.canada.ca/en/services/taxes.html">Taxes</a></h3>
 				<p>Income tax, payroll, GST/HST,  contribution limits, tax credits, charities</p>
 			</div>
 			<div class="col-lg-4 col-md-6">
-				<h3><a href="https://www.canada.ca/en/services/environment.html">Environment and natural resources</a></h3>
-				<p>Weather, climate, agriculture, wildlife, pollution, conservation, fisheries</p>
-			</div>
-			<div class="col-lg-4 col-md-6">
-				<h3><a href="https://www.canada.ca/en/services/defence.html">National security and defence</a></h3>
-				<p>Military, transportation and cyber security, securing the border, counter-terrorism</p>
-			</div>
-			<div class="col-lg-4 col-md-6">
-				<h3><a href="https://www.canada.ca/en/services/culture.html">Culture, history and sport</a></h3>
-				<p>Arts, media, heritage, official languages, national identity and funding</p>
-			</div>
-			<div class="col-lg-4 col-md-6">
-				<h3><a href="https://www.canada.ca/en/services/policing.html">Policing, justice and emergencies</a></h3>
-				<p>Safety, justice system, prepare for emergencies, services for victims of crime</p>
-			</div>
-			<div class="col-lg-4 col-md-6">
-				<h3><a href="https://www.canada.ca/en/services/transport.html">Transport and infrastructure</a></h3>
-				<p>Aviation, marine, road, rail, dangerous goods, infrastructure projects</p>
+				<h3><a href="https://www.canada.ca/en/services/health.html">Health</a></h3>
+				<p>Food, nutrition, diseases, vaccines, drugs, product safety and recalls</p>
 			</div>
 			<div class="col-lg-4 col-md-6">
-				<h3><a href="https://international.gc.ca/world-monde/index.aspx?lang=eng">Canada and the world</a></h3>
-				<p>Foreign policy, trade agreements, development work, global issues</p>
+				<h3><a href="https://www.canada.ca/en/services/environment.html">Environment and natural resources</a></h3>
+				<p>Weather, climate, agriculture, wildlife, pollution, conservation, fisheries</p>
 			</div>
 			<div class="col-lg-4 col-md-6">
 				<h3><a href="https://www.canada.ca/en/services/finance.html">Money and finances</a></h3>
 				<p>Personal finance, credit reports, fraud protection, paying for education</p>
 			</div>
-			<div class="col-lg-4 col-md-6">
-				<h3><a href="https://www.canada.ca/en/services/science.html">Science and innovation</a></h3>
-				<p>Scientific research on health, environment and space, grants and funding</p>
-			</div>
+		</div>
+		<div>
+			<a class="btn btn-call-to-action" href="https://www.canada.ca/en/services.html">All services</a>
```

Comment:
I didn't say to use the call-to-action, my point was more do an implementation similar like the call to action which will achieve the desired behaviour and visual effect as illustrated by their tested prototype.

If it is easier, you can extract each desired behaviour/style from their prototype and log that as individual Technical Requirement in the Jira request like Sous toute reserve et seulement pour présenter le principe:  "TR-1: button style: border 2px, padding 3px...." "TR-2: button behaviour: ensure the button always remain on its own row" (I didn't check the desired effect of the use of max-width)